code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -176,6 +191,7 @@
 	id = "soymilk"
 	description = "An opaque white liquid made from soybeans."
 	color = "#DFDFC7" // rgb: 223, 223, 199
+	taste_description = "soy milk"
```

Comment:
"it tastes like mil.... wait nevermind, that certainly isn't the milk i'm used to!"